src/shared/views/Login.js

```diff
@@ -199,25 +200,27 @@ class _Login extends React.Component<Props, State> {
         const { firebase, navigation, t } = this.props;
         const { email, password, username } = this.state;
         const parent = this;
-        if (username !== null && username.length < MIN_USERNAME_LENGTH) {
+
+        if (username !== null && username.indexOf('@') !== -1) {
```

Comment:
We do not need to check for '@' as it is already covered by the regex characters validation.